resin-init-flasher: use halt instead of shutdown

This board does not supports normal power off, rebooting instead.
So call halt instead.

Index: 1.0-r2/resin-init-flasher
===================================================================
--- 1.0-r2.orig/resin-init-flasher
+++ 1.0-r2/resin-init-flasher
@@ -490,7 +490,7 @@ report_progress 100 "Post-Provisioning"
 # This is achieved because the shutdown command is not part of the initramfs
 if command -v shutdown; then
     info "Shutting down ..."
-    shutdown -h now
+    halt
 elif command -v reboot; then
     info "Rebooting..."
     reboot -f
